courses/management/commands/bulk_load_topics.py

```diff
@@ -0,0 +1,98 @@
+"""Management command to load bulk load topics from a CSV into system
+
+Arguments:
+* --file <filename> - Path to the CSV file containing the topics
+
+--file must be a valid CSV file. The command assumes that the file would have a header row containing all the topics/subtopics.
+The sample file format can be seen below:
+
+topic       | topic         | topic
+subtopic    | subtopic      | subtopic
+subtopic    | subtopic      | subtopic
+
+"""
+
+import csv
+
+from django.core.management import BaseCommand, CommandError
+
+from courses.models import CourseTopic
+
+
+def create_topic(name=None, parent=None):
+    """Helper method to create topics based on topic/subtopic relationship"""
+    name = name.strip()
+    if not parent:
+        return CourseTopic.objects.get_or_create(name=name)
+    parent = parent.strip()
+    parent_topic = CourseTopic.objects.get(name=parent, parent__isnull=True)
```

Comment:
There may be a scenario where the child topic is a parent of another topic. In that case, this will raise an error that the `courses.models.CourseTopic.DoesNotExist: CourseTopic matching query does not exist.`

CSV will be like:
```
Topic, Subtopic
Subtopic, anotherSubTopic
```

Is this a valid assumption?